.github/workflows/main_push.yml

```diff
@@ -1,17 +1,11 @@
 name: CI
 
+permissions: write-all
```

Comment:
Maybe a bit too permissive here. It's better to set only the strict needed permissions.

Reply:
What would be for you the strictly needed permissions?

Reply:
Done in https://github.com/Activiti/Activiti/pull/4529/commits/26f530c995e018031d562fa585dd8b65c12c7de0